### DISCUSSION TITLE
org agenda get not restored

### DISCUSSION THREAD

**Discussion by hokreb** [alphapapa/activities.el#94] [Q&A] [answered]
I switch regularly to a frame with two windows, left my projects org file and right my org agenda view. Obviously, as org agenda view can not be saved as bookmarks, activities can't restore it out of the box. Are there hooks available, by which I can check eg the name of the activity to be opened so the missing org agenda view can be created before the activity is actually restored?

**Comment by alphapapa** [comment]
Have you looked in the customization options or source code?

Also, FWIW, Org QL view buffers can be bookmarked and restored (obviously they don't have all of the features of Org Agenda buffers, but they generally meet my needs).

**Reply by hokreb** [reply]
You are right, I havn't looked into the code :-), I found the "activities-before-resume-functions", which would be the one to be useful for my task. This function seems to be never called

**Reply by jdtsmith** [reply, marked as answer]
Did you see [this comment](https://github.com/alphapapa/activities.el/discussions/64#discussioncomment-8966550)?

**Reply by alphapapa** [reply]
> You are right, I havn't looked into the code :-), I found the "activities-before-resume-functions", which would be the one to be useful for my task. This function seems to be never called

Thanks.  See https://github.com/alphapapa/activities.el/issues/99
